crypto/aead/src/lib.rs

```diff
@@ -98,19 +98,6 @@ pub fn get_encryption_key(password: &str, salt: &str) -> Result<LessSafeKey> {
     Ok(LessSafeKey::new(key))
 }
 
-/// Memory-hard Argon2 key stretching for password-based authentication
-///
-/// * `password` - Strong user-created password
-/// * `salt` - B64 encoded nonce between 4 and 64 bytes
```

Comment:
Removing some dead code because it will cause tests to fail if we run all the tests with real fixtures eventually.